pootle/constants.py

```diff
@@ -6,4 +6,4 @@
 # or later license. See the LICENSE file for a copy of the license and the
 # AUTHORS file for copyright and authorship information.
 
-VERSION = (2, 8, 0, 'beta', 3)
+VERSION = (2, 8, 0, 'beta', 4)
```

Comment:
Note for future, we need to change this up after each release as we've been working on beta4 up until now and this just confuses people.